packages/adminrouter/extra/src/includes/http/common.conf

```diff
@@ -1,9 +1,87 @@
-# Enable VTS module for metrics
-# Set the shared memory zone size to 32MB to be sure to
-# contain all metrics in the VTS module configuration.
-# If the VTS module configuration enables dynamic metric
-# keys this needs to be increased to accommodate for that.
-vhost_traffic_status_zone shared:vhost_traffic_status:32m;
+# Enable VTS module for metrics.
+#
+# The VTS module uses shared memory to store metrics data.
+# In particular, we store metrics data generated by the
+# ``vhost_traffic_status_filter_by_set_key`` directive.
+#
+# If this memory is used, Admin Router does not crash, but errors are shown
```

Comment:
Given the more specific information two paragraphs down, this sentence could be removed.